### ISSUE TITLE
Limit the number of checkboxes that can be checked

### ISSUE THREAD

**Issue by Odepia** [gyrocode/jquery-datatables-checkboxes#63]
Is there a way to limit the number of checkboxes that can be checked?
Let's say that I want to limit the selection to 2 checkbox.

Thanks

**Comment by Instabash** [comment]
> Is there a way to limit the number of checkboxes that can be checked?
> Let's say that I want to limit the selection to 2 checkbox.
> 
> Thanks

Hey, have you found a solution to this?

**Comment by Odepia** [comment]
> Hey, have you found a solution to this?

No, I haven't! I had to go through another solution to do what I wanted.
But, if you have a really one, let me know, I am still interesting about a solution, even after 2 years!

**Comment by mpryvkin** [comment]
There could be a couple ways of doing this, let me experiment with it and I can produce an example.

**Comment by mpryvkin** [comment]
This [example](https://jsfiddle.net/1nyjdsvx/) is the closest I could come up with without modifying the Checkboxes plug-in. I had to disable "select all" control because it selects all checkboxes without limitation. I may need to think what kind of modifications are needed.

**Comment by Odepia** [comment]
Thank you very much for taking the time to answer me and produce a working example!, Michael! 
It's really appreciated!
